README.md

```diff
@@ -178,6 +178,24 @@ inherit_from:
   - node_modules/@prettier/plugin-ruby/rubocop.yml
 ```
 
+## Editor Integration
+
+### VS Code
+
+- Add the npm `@prettier/plugin-ruby` package to your project as described above.
+- Install the [Prettier - Code Formatter](https://github.com/prettier/prettier-vscode) extension.
+- Configure in your `settings.json`:
+  ```json
+  {
+    "[ruby]": {
+      "editor.defaultFormatter": "esbenp.prettier-vscode",
+      "editor.formatOnSave": true // optional
```

Comment:
`// optional` isn't valid json and messes up syntax highlighting

Reply:
It's valid in VS Code's settings config but I'll remove it.